test/e2e/runner/load.go

```diff
@@ -172,10 +160,21 @@ func createTxBatch(ctx context.Context, txCh chan<- types.Tx, testnet *e2e.Testn
 					panic(fmt.Sprintf("Failed to generate tx: %v", err))
 				}
 
-				select {
-				case txCh <- tx:
-				case <-ctx.Done():
-					return
+				var nodeIndices []int
+				if testnet.LoadNumNodesPerTx <= 1 {
+					// Pick one random node to send the transaction.
+					nodeIndices = []int{cmtrand.Intn(len(testnet.Nodes))}
```

Comment:
Given the random generator, are we sure it is not going to pick the same node every time? Or at least very often? Should we in this case not use an increasing list of idxs perhaps?

Reply:
Isn't it a very similar behaviour to what we have now? Currently we have many readers (one per node and per connection) on the same channel waiting for a transaction. 

From the golang [spec](https://go.dev/ref/spec#Select_statements):
> 2. If one or more of the communications can proceed, a single one that can proceed is chosen via a uniform pseudo-random selection. Otherwise, if there is a default case, that case is chosen. If there is no default case, the "select" statement blocks until at least one of the communications can proceed.

Reply:
I guess you are right. The only other concern is what I wrote below, if one of those nodes is supposed to be skipped  (because it has `send_no_load` to true.